### PR TITLE
US16T04 middleware comment

### PR DESCRIPTION
**Pull request by enerisan**

### CODE REVIEW

**Review by yannduhamel — DISMISSED** [review]

**Review by AnneDAMIEN — APPROVED** [review]

**Review by Pierre-Voisin — APPROVED** [review]

**Review by angelique888 — APPROVED** [review]

**Review by Pierre-Voisin — APPROVED** [review]

**Review by AnneDAMIEN — DISMISSED** [review]

**Review by Pierre-Voisin — APPROVED** [review]

**Review by ioayoub — DISMISSED** [review]

**Review by AnneDAMIEN — DISMISSED** [review]

**Review by angelique888 — DISMISSED** [review]

**Review by yannduhamel — DISMISSED** [review]

**Review by Pierre-Voisin — APPROVED** [review]

File: server/app/middlewares/validateCommentSchema.js [hunk 1]
```diff
@@ -3,10 +3,10 @@ const { z } = require("zod");
 const commentSchema = z.object({
   description: z
     .string({
-      invalid_type_error: "Votre date n´est pas valide",
+      invalid_type_error: "Votred description n´est pas valide",
```
Comment (ioayoub) [inline server/app/middlewares/validateCommentSchema.js:6]: typo

File: server/app/middlewares/validateCommentSchema.js [hunk 2]
```diff
@@ -3,10 +3,10 @@ const { z } = require("zod");
 const commentSchema = z.object({
   description: z
     .string({
-      invalid_type_error: "Votre date n´est pas valide",
+      invalid_type_error: "Votred description n´est pas valide",
     })
     .min(2, {
-      message: "Votre commentaire doit contenir au minimun 3 charactères",
+      message: "Votre commentaire doit contenir au minimun 2 charactères",
```
Comment (ioayoub) [inline server/app/middlewares/validateCommentSchema.js:9]: typo in last word

**Review by ioayoub — CHANGES_REQUESTED** [review]

File: server/app/middlewares/validateCommentSchema.js (hunk 2 above)
Comment (enerisan) [reply server/app/middlewares/validateCommentSchema.js]: done

File: server/app/middlewares/validateCommentSchema.js (hunk 1 above)
Comment (enerisan) [reply server/app/middlewares/validateCommentSchema.js]: done

**Review by ioayoub — APPROVED** [review]

**Review by yannduhamel — APPROVED** [review]

**Review by Pierre-Voisin — APPROVED** [review]

**Review by angelique888 — APPROVED** [review]

**Review by AnneDAMIEN — APPROVED** [review]

**Pull request merged**